### PR TITLE
Put dtfj, dtfjview and traceformat classes in tools.jar

### PR DESCRIPTION
**Pull request by groeges**
Signed-off-by: Steve Groeger <GROEGES@uk.ibm.com>

### CODE REVIEW

**Comment by groeges** [comment]
@andrew-m-leonard As per @pshipton I have moved the classes for these jars to the tools.jar instead of having them as separate jars in the lib/ext directory. 
This PR does just that.  Please can you review and merge if OK.

File: jdk/make/CreateJars.gmk [hunk 1]
```diff
@@ -590,7 +555,20 @@ $(eval $(call SetupArchive,BUILD_TOOLS_JAR, , \
         META-INF/services/com.sun.jdi.connect.spi.TransportService \
         META-INF/services/com.sun.tools.attach.spi.AttachProvider \
         META-INF/services/com.sun.tools.internal.ws.wscompile.Plugin \
-        META-INF/services/com.sun.tools.internal.xjc.Plugin, \
+        META-INF/services/com.sun.tools.internal.xjc.Plugin \
+        com/ibm/jvm/Debug.class \
```
Comment (pshipton) [inline jdk/make/CreateJars.gmk]: Where do these classes come from? I don't find them.

**Comment by pshipton** [comment]
Do the executables still work with these changes? I expected a change so they look in the tools.jar, which isn't on the classpath by default.

File: jdk/make/CreateJars.gmk (hunk 1 above)
Comment (groeges) [reply jdk/make/CreateJars.gmk]: Which classes are you referring to? If you mean the META-INF/services/.... ones, then these were in that make file already and I presumed they are related to OpenJDK8. If you mean the com/ibm/jvm/... ones then these were listed in the icl-build.mk file that I was pointed to which was previously used to build the tracefomat.jar. These classes do not exist as separate java files (which caused me some issues initially) but they are classes that are defined in other java classes (which I find very strange!!!).

**Comment by groeges** [comment]
In my build they still seem to work. I run `./build/linux-x86_64-normal-server-release/images/j2sdk-image/bin/traceformat` and that works.
Hmm, if I run `./build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/bin/traceformat` it doesn't as it can't find the class, as you expected. 
Will try and see if I can get these working!!!

**Comment by pshipton** [comment]
It is expected that traceformat doesn't work in the jre image, as the jre image doesn't contain tools.jar. The executables should only be present in the bin directory and only part of the sdk image. They should not be present in the jre/bin directory and part of the jre image. This is similar to other tools like javac, rmic, etc.

File: jdk/make/CreateJars.gmk (hunk 1 above)
Comment (pshipton) [reply jdk/make/CreateJars.gmk]: Ok I see, they are defined in the com.ibm.jvm.TraceFormat.java file.

**Pull request merged**